merlin/config/configfile.py

```diff
@@ -141,13 +169,43 @@ def set_username_and_vhost(config: Dict):
         config["broker"]["vhost"] = vhost
 
 
-def get_config(path: Optional[str]) -> Dict:
+def get_default_config() -> Dict:
+    """
+    Creates a minimal default configuration for local mode.
+
+    Returns:
+        Dict: A configuration dictionary with essential default values.
+    """
+    default_config = {
+        "broker": {
+            "username": getpass.getuser(),
+            "vhost": getpass.getuser(),
```

Comment:
an odd choice for a default

Reply:
I was unsure whether to leave this or not. We can probably just set the values here to "default" as this default config is never actually being connected to. It's necessary so that the CONFIG object can be loaded in properly 
as it needs a configuration of this format, but the values in it shouldn't matter.